templates/huawei_vrp_display_lldp_neighbor.textfsm

```diff
@@ -0,0 +1,76 @@
+Value Required LOCAL_INTERFACE (\S+)
+Value CHASSIS_ID (.+?)
+Value MANUFACTURER_NAME (.*)
```

Comment:
Update the captures of `.*` to use `.*?` and add `\s*$$` at the end of the line to ensure that `.*` doesn't capture a trailing space

Reply:
Updated